src/04-modality-specific-files/10-microscopy.md

```diff
@@ -0,0 +1,425 @@
+| BodyPart                    | RECOMMENDED           | [string][]                             | Body part of the organ / body region scanned. From [DICOM Body Part Examined](http://dicom.nema.org/medical/dicom/current/output/chtml/part16/chapter_L.html#chapter_L) (for example: `"BRAIN"`)".                                                                                                                                                                                                                                                                                                                         |
+| BodyPartDetails             | RECOMMENDED           | [string][]                             | If needed, additional details about body part or location (for example: `"corpus callosum"`).                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| BodyPartDetailsOntology     | OPTIONAL              | [string][]                             | If applicable, information about the ontology used for specifying BodyPartDetails (for example: `"UBERON (https://www.ebi.ac.uk/ols/ontologies/uberon)"`).                                                                                                                                                                                                                                                                                                                                                                 |
+| SampleEnvironment           | RECOMMENDED           | [string][]                             | Indicate if the sample is imaged `"invivo"`, `"exvivo"` or `"invitro"`.                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
```

Comment:
Is there a reason to drop the spaces? And is this a controlled vocabulary, or are these examples? Assuming controlled vocabulary:

```suggestion
| SampleEnvironment           | RECOMMENDED           | [string][]                             | Environment in which the sample was imaged. One of: `"in vivo"`, `"ex vivo"` or `"in vitro"`.                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
```

Reply:
Good catch, thank you! It comes from an old habit of mine to avoid spaces in labels, filenames, etc 😅
As a string, we can add the space back. They were intended as controlled vocabulary, I'll make sure we had a validation for that as well.